guildwars2/guild/sync.py

```diff
@@ -195,10 +231,31 @@ def check(m):
         except Exception as e:
             return None
 
+    async def add_member_to_role(self, role, member, guild):
+        try:
+            results = await member.add_roles(
```

Comment:
add_roles doesnt return anything, will always be None unless there's an exception